apps/sample-configs/renderManager.direct.portrait.json

```diff
@@ -6,10 +6,10 @@
         "directModeEnabled": true,
         "directDisplayIndex": 0,
         "directHighPriorityEnabled": true,
-        "numBuffers": 1,
-        "verticalSyncEnabled": false,
+        "numBuffers": 2,
```

Comment:
just curious - is front-buffer rendering even expected to work in the current RenderManager? I haven't been able to get it to work without tearing so far.